src/redux/ConnectorState/ConnectorUrlsSlice.ts

```diff
@@ -0,0 +1,35 @@
+import { createSlice, PayloadAction } from '@reduxjs/toolkit'
+
+import { RootState } from '/redux/store'
+
+export interface ConnectorUrlsState {
+  urls: string[]
+}
+
+const initialState: ConnectorUrlsState = {
+  urls: []
+}
+
+export const connectorUrlsSlice = createSlice({
+  name: 'connectorUrls',
+  initialState,
+  reducers: {
+    updateList: (state, action: PayloadAction<string>) => {
+      state.urls = [...state.urls, action.payload]
```

Comment:
Maybe we can dedupe urls?

Reply:
after fixing the other issue, I'm not sure it's very important (shouldn't happen too)